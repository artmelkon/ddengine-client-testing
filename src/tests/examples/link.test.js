'use strict';

import Link from './Link';
import renderer from 'react-test-renderer';

test('Link renders correctly', () =>
{
  const tree = renderer
    .create(<Link page='https://www.facebook.com'>Facebook</Link>)
    .toJSON();
  expect(tree).toMatchSnapshot()
});

it('renders as an anchor when no page is set', () =>
{
  const tree = renderer.create(<Link>Facebook</Link>).toJSON();
  expect(tree).toMatchSnapshot();
})
