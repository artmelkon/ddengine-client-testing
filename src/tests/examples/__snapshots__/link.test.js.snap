// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Link renders correctly 1`] = `
<a
  className="normal"
  href="https://www.facebook.com"
  onMouseEnter={[Function]}
  onMouseLeave={[Function]}
>
  Facebook
</a>
`;

exports[`renders as an anchor when no page is set 1`] = `
<a
  className="normal"
  href="#"
  onMouseEnter={[Function]}
  onMouseLeave={[Function]}
>
  Facebook
</a>
`;
